protobuf/v1/schema.proto

```diff
@@ -0,0 +1,272 @@
+syntax = "proto3";
+
+package schema.v1;
+
+option go_package = "schema/service/v1";
+
+import "google/protobuf/struct.proto";
+import "google/api/annotations.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+message ResolveBooleanRequest {
+    string flag_key = 1;
+    google.protobuf.Struct context = 2;
+    bool default_value = 3;
+}
+message ResolveBooleanResponse {
+    bool value = 1;
+    string reason = 2;
+    string variant = 3;
+}
+
+message ResolveStringRequest {
+    string flag_key = 1;
+    google.protobuf.Struct context = 2;
+    string default_value = 3;
+}
+message ResolveStringResponse {
+    string value = 1;
+    string reason = 2;
+    string variant = 3;
+}
+
+message ResolveNumberRequest {
+    string flag_key = 1;
+    google.protobuf.Struct context = 2;
+    float default_value = 3;
+}
+message ResolveNumberResponse {
+    float value = 1;
```

Comment:
Should we consider oneOf here to support ints and floats? Not sure it's 100% needed, float may be good enough... something to consider.